src/ctap/hid/mod.rs

```diff
@@ -421,6 +421,7 @@ mod test {
     #[test]
     fn test_split_assemble() {
         for payload_len in 0..7609 {
+            let mut env = TestEnv::new();
```

Comment:
Do we need to create this in the loop? I would have put it outside the loop.

Reply:
Done